src/fenicsxconcrete/experimental_setup/compression_cylinder.py

```diff
@@ -99,6 +101,7 @@ def __init__(self, parameters: dict[str, pint.Quantity]=None):
         p.update(parameters)
 
         super().__init__(p)
+        self.logger = logging.getLogger("fenicsxconcrete.experimental_setup.compression_cylinder.CompressionCylinder")
```

Comment:
I usually use "self.logger = logging.getLogger(__name__)" that should be the same here and is much shorter to write ;)

Reply:
I meant __name__ not bold

Reply:
you mean `__name__` ? yes that is a good idea. thanks :)

Reply:
_ _ name _ _

Reply:
it will be the name of the module and not the class though, but I guess this is fine.